src/ManagedServices/ManagedServices/Commands/GetAzureRmManagedServicesDefinition.cs

```diff
@@ -22,25 +22,15 @@ namespace Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Commands
     [Cmdlet(
     VerbsCommon.Get,
     Microsoft.Azure.Commands.ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ManagedServicesDefinition",
-    DefaultParameterSetName = ByIdParameterSet), OutputType(typeof(PSRegistrationDefinition))]
+    DefaultParameterSetName = ByNameParameterSet), OutputType(typeof(PSRegistrationDefinition))]
     public class GetAzureRmManagedServicesDefinition : ManagedServicesCmdletBase
     {
-        ////protected const string DefaultParameterSet = "Default";
-        protected const string ByResourceIdParameterSet = "ByResourceId";
-        protected const string ByIdParameterSet = "ById";
+        protected const string ByNameParameterSet = "ByName";
 
-        [Parameter(ParameterSetName = ByIdParameterSet, Mandatory = false, HelpMessage = "The registration definition identifier.")]
+        [Parameter(ParameterSetName = ByNameParameterSet, Mandatory = false, HelpMessage = "The unique name of the Registration Definition.")]
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
-        [Parameter(
-            ParameterSetName = ByResourceIdParameterSet,
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The full qualified resource id of registration definition.")]
-        [ValidateNotNullOrEmpty]
-        public string ResourceId { get; set; }
```

Comment:
removing mandatory parameter is breaking change and is not allowed in non-breaking change release